woodwork/tests/accessor/test_statistics.py

```diff
@@ -830,3 +833,262 @@ def test_get_numeric_value_counts_in_range(input_series, expected):
     column = input_series
     top_values = _get_numeric_value_counts_in_range(column, range(4))
     assert top_values == expected
+
+
+def test_calculate_iqr_bounds_with_quantiles(outliers_df_pandas):
+    expected_low = 8.125
+    expected_high = 83.125
+
+    series = outliers_df_pandas['has_outliers']
+
+    q1, q3 = np.percentile(series, [25, 75])
+    quantiles = {0.25: q1, 0.75: q3, 0.0: 100000}
+
+    low, high = _calculate_iqr_bounds(outliers_df_pandas['has_outliers'], quantiles=quantiles)
```

Comment:
It seems like the quantiles that are being passed into `_calculate_iqr_bounds` here will be ignored since the function is designed to calculate them from the given series if one is provided.  Is this test case meant to probe that behavior?  If so, it can maybe be a bit more targeted and we probably wouldn't need to test it twice.  If we're not trying to test that behavior, was it maybe forgotten in this case that the function behaves that way?

Reply:
whoopse, definitely a case of me forgetting how this function behaves - will remove the series!

Reply:
by removing the `outliers_dict` method, there's no longer a reason to have separate `series` and `quantiles` params, so I've removed the helper function and the logic now lives in `_get_box_plot_info_for_column` since the logic for whether we need quantiles or not has to exist earlier in the box plot logic anyway.